velox/expression/ConjunctExpr.cpp

```diff
@@ -135,13 +131,6 @@ void ConjunctExpr::evalSpecialForm(
       context.swapErrors(errors);
     }
 
-    // AND: reduce finalSelection to activeRows unless it has been fixed by IF
-    // or OR above.
-    VarSetter finalSelectionAnd(
```

Comment:
Is this not needed anymore?

Reply:
Setting finalSelection without setting isFinalSelection to false is inconsequential as it will be overridden whenever final selection needs to be activated (that is, when isFinalSelection is set to false and finalSelection is set to a selectivity vector)
Moreover, if the idea behind this line was to reduce the finalSelection, it will automatically be reduced as the set of inputRows gets reduced as we progress while processing AND's children, so when we encounter a situation where we need to activate finalSelection, the current inputRows are used to set it.

Reply:
Thank you for the clarification.